ui/src/components/system/Settings.vue

```diff
@@ -243,6 +243,29 @@
             >{{errors.OtpCockpit.message}}</span>
           </div>
         </div>
+        <div
+          v-if="view.otpIsLoaded && otp.OtpStatus"
+          :class="['form-group', errors.OtpSshd.hasError ? 'has-error' : '']"
+        >
+          <label
+            class="col-sm-2 control-label"
+            for="textInput-modal-markup"
+          >{{$t('settings.OtpSshd_status')}}</label>
```

Comment:
Can we add an inline-doc here?
The tool tip should state something like:
```
2FA is available only for password authentication.
Authentication with public keys will not be affected by 2FA.
```

Reply:
@gsanchietti done